components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/mappings/AdvancedThrottlePolicyMappingUtil.java

```diff
@@ -0,0 +1,128 @@
+/*
+ *
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
extra new line and wrong year.  are these auto-generated classes? if yes, we'll have to fix the code generator.